core/src/main/scala/monocle/std/Tree.scala

```diff
@@ -2,6 +2,8 @@ package monocle.std
 
 import monocle.function._
 import monocle.{Iso, Lens}
+import monocle.function.all._
+import monocle.std.stream._
```

Comment:
I think it is used to bring in scope the `HeadOption` instance for `Stream` but you raise a good point, I think there is a compiler option to disallow unused import.